### ISSUE TITLE
C420 Cameras and H200 Hub unavailable after update to 5.3.0-5.3.3

### ISSUE THREAD

**Issue by antstyl** [JurajNyiri/HomeAssistant-Tapo-Control#392]
### Description

Right after update to 5.3.0 and 5.3.1 the two cameras C420 and the hub H200 dont work anymore.
In the Integrations section, it says "Failed Setup, will retry" 

In the logs I get this:

Logger: custom_components.tapo_control
Source: custom_components/tapo_control/__init__.py:691
Integration: Tapo: Cameras Control ([documentation](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control), [issues](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/issues))
First occurred: 12:19:18 μ.μ. (4 occurrences)
Last logged: 12:21:55 μ.μ.

Unable to connect to Tapo: Cameras Control controller: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

### Reproduction Steps

1. Go to Devices & Services
2. Go to Tapo: Cameras Control

### Expected behavior

Cannot see battery status (as I used to) and cannot change anything as all are unavailable

### Device Firmware

1.2.21 Build 20230731 rel.51223 (HUB) and 1.2.11 Build 20230726 rel.66682 (Cameras)

### Using stream component

Yes

### Does camera work via official integrations?

Yes

### Camera has all attributes filled out in developer tools

Yes

### HASS Environment

VM in Proxmox

### Search for similar issues

Yes

**Comment by JurajNyiri** [comment]
Hi, wait for the retries and try rebooting the camera, it might be overloaded.

**Comment by JurajNyiri** [comment]
IF, and **only** IF it does not come up after rebooting camera and waiting for at least 10 minutes, try installing https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.3.0.beta.3 and let me know how it goes in either case.

**Comment by JurajNyiri** [comment]
If issue persists, please enable debug logs.

```
logger:
  default: critical
  logs:
    custom_components.tapo_control: debug
```

Upload the full log here.

**Comment by antstyl** [comment]
After at least 10 mins and reboot of Home assistant and of Hub, problem still persists.

Will try to install the older version (just copy and replace the files in config/custom_components folder? ) and let you know.

For the records, if this has any meaning to you, there is an sd card in the Hub.

**Comment by JurajNyiri** [comment]
Yes you can also install betas via HACS.

**Comment by JurajNyiri** [comment]
Also rollback to last working version to verify it is not a camera issue please.

**Comment by antstyl** [comment]
Just rolled back to previous state through back up. In 5.2.4 it works ok.

Do you want me to update step by step in order to see where the problem starts?

These are the versions I have in HACS.
![image](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/assets/6030798/e7608b8a-6034-44a5-855d-4f824a2965aa)

**Comment by JurajNyiri** [comment]
Please try https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.3.0.beta.3 and https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.3.0.beta.5. You will need to install it manually if you do not see it in HACS.

**Comment by antstyl** [comment]
Ok will do it this night and let you know.

**Comment by JurajNyiri** [comment]
You can also install `pytapo` library on python. `python3 -m pip install pytapo` and see if this works:


```
from pytapo import Tapo

# Camera
user = "admin"  # admin user
password = ""  # cloud password
host = "192.168.100.70"  # ip of the camera, example: 192.168.1.52

tapo = Tapo(host, user, password, password)

print(tapo.getMost())
```

**Comment by JurajNyiri** [comment]
Also please test with [5.3.3](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.3.3).

**Comment by antstyl** [comment]
Problem persists after update to 5.3.3 and one more Camera (C310) became unavailable.

Also new error in logs, as described in #393

**Comment by JurajNyiri** [comment]
Follow all steps above otherwise I cannot help you.

**Comment by antstyl** [comment]
Ok I will, just wanted to test first the 5.3.3 and then go back.

Will keep you posted. thanks.

**Comment by CAP-Team** [comment]
5.3.0 works fine for my C310, 5.3.3 doesn't work.

**Comment by JurajNyiri** [comment]
Thank you for testing @antstyl!

Closing as this issue is now fixed in [5.3.4](https://github.com/JurajNyiri/HomeAssistant-Tapo-Control/releases/tag/5.3.4).
